src/components/landing-page/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { LANDING_PAGE_FEATURES } from "@/lib/constants/misc";
+import { Button } from "../ui/button";
+import Image from "next/image";
+import { grayBlurDataValue } from "@/lib/utils";
+
+const LandingPage = () => {
+  return (
+    <>
+      <div className="bg-[url('/images/app_landing.jpg')] bg-center md:h-[400px] bg-cover rounded-[12px] p-[40px]">
```

Comment:
I would suggest to use next image for the optimisation

Reply:
@karmatenzi  since the image is for presentation purpose only, not necessary to use Image but may need to reduce image size keeping the required quality